tools/generate-operator-patches.ts

```diff
@@ -57,19 +58,26 @@ const AliasMethodOverrides = {
 };
 
 function generateNewOperatorFileContents (op:OperatorWrapper): OperatorWrapper {
+  var header = `/**
+ * Everything in this file is generated by the 'tools/generate-operator-patches.ts' script.
+ * Any manual edits to this file will be lost next time the script is run.
+ **/`;
   var baseObject = op.isExtended ? 'observableProto' : 'Observable';
   var optPrototype = op.isStatic || op.isExtended ? '' : 'prototype.';
-  var levelsUp = op.isExtended ? '../../../' : '../../';
-  var imports = `import {Observable} from '${levelsUp}Observable';
-import {${op.exportedFnName}} from '${levelsUp}operator/${op.path.replace('.ts','')}';
-${op.isExtended ? 'import {KitchenSinkOperators} from \'../../../Rx.KitchenSink\';' : ''}`;
-  var extendedProto = `${op.isExtended ? 'const observableProto = (<KitchenSinkOperators<any>>Observable.prototype);' : ''}`
+  var imports = `import {Observable} from '../../Observable';
+import {${op.exportedFnName}} from '../../operator/${op.path.replace('.ts','')}';
+${op.isExtended ? 'import {KitchenSinkOperators} from \'../../Rx.KitchenSink\';' + '\n': ''}`;
+  var extendedProto = 'const observableProto = (<KitchenSinkOperators<any>>Observable.prototype);'
   var patch = op.aliases.map((alias) => {
     return `${baseObject}.${optPrototype}${alias} = ${op.exportedFnName};`;
   }).join('\n');
+  var typescriptHack = 'export var _void: void;'
```

Comment:
Good idea. Since this hack is spread over almost all files (not only the auto-generated ones), I'll add a comment in this script instead of injecting the comment in the final patching files.